### DISCUSSION TITLE
How do I interact with the web page?

### DISCUSSION THREAD

**Discussion by gerroon** [n4ze3m/page-assist#52] [Q&A]
Hi

How do I interact and summarize web pages? I open the side bar while a page is open then I type "summarize the page" but I am getting 

"I apologize, but there is no page for me to summarize. It seems that we started a conversation without a specific topic or text to work with. If you'd like to share some information or ask a question, I'm here to help"

thanks

**Comment by n4ze3m** [comment]
Hey, sorry about it. Usually, it works, but some websites are having issues. I will look into it

![image](https://github.com/n4ze3m/page-assist/assets/39720973/29314b58-8fea-4c90-bcbd-a6e959b4c8e0)
